### ISSUE TITLE
Regression in 0.5.0 using reference links to files that exist

### ISSUE THREAD

**Issue by adamvoss** [benbalter/jekyll-relative-links#36]
The following file works fine (renders the link) with 0.5.0

```
---
layout: page
title: Examples
---

[link][link]

[link]: does_not_exist
```

However, if you create `does_not_exist` as an empty file, it renders as `[link][link] [link]: /does_not_exist`.

However on 0.4.1 it would render a link regardless of whether the file existed or not.

**Comment by qwtel** [comment]
This could be related to #31. Does it work with links that do exist? Could you try it with the master branch of this repository (where #31 is already fixed), e.g. in your Gemfile:

```ruby
gem "jekyll-relative-links", git: "git@github.com:benbalter/jekyll-relative-links.git"
```

**Comment by adamvoss** [comment]
@qwtel Tested and confirmed that it is fixed in the current HEAD (77a12a2).  Thanks for fixing this!  I hope they release it soon-ish :wink:

**Comment by benbalter** [comment]
Thanks for test. Released https://github.com/benbalter/jekyll-relative-links/releases/tag/v0.5.1 with the fix (and opened https://github.com/github/pages-gem/pull/494 to bump the dependency).
